integrated_channels/sap_success_factors/utils.py

```diff
@@ -266,3 +320,13 @@ def transform_language_code(code):
     language_family = SUCCESSFACTORS_OCN_LANGUAGE_CODES[language_code]
     language_name = language_family.get(country_code, language_family['_'])
     return language_name
+
+
+def safe_extract_key(data, key, default=''):
```

Comment:
@mattdrayer that makes sense, would you recommend another edx owned library or just a new file in this package? also I'm wondering if there is something like this that already exists.